### PR TITLE
Add GraphQL custom fragments for Gatsby Image

### PR DESCRIPTION
**Pull request by valse**
As subject I added some GraphQL fragments to customize the traced SVG effect color and for improve the reusability for the Gatsby Image plugin.

### CODE REVIEW

File: src/lib/fragments.js [hunk 1]
```diff
@@ -0,0 +1,18 @@
+import {graphql} from 'gatsby'
+
+export const bannerImage = graphql`
```
Comment (kentcdodds) [inline src/lib/fragments.js:3]: Super! So does Gatsby look for this file and load it automatically?

File: src/lib/fragments.js [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+  fragment bannerImage640 on File {
+    childImageSharp {
+      fluid(maxWidth: 640, traceSVG: {color: "#573ede"}) {
+        ...GatsbyImageSharpFluid_withWebp_tracedSVG
+      }
+    }
+  }
+  fragment bannerImage720 on File {
+    childImageSharp {
+      fluid(maxWidth: 720, traceSVG: {color: "#573ede"}, quality: 75) {
```
Comment (kentcdodds) [inline src/lib/fragments.js:13]: Is it possible to make fragments accept an argument or something?

**Review by kentcdodds — APPROVED** [review]
Awesome! Thanks 💯

File: src/lib/fragments.js (hunk 1 above)
Comment (valse) [reply src/lib/fragments.js:3]: Yes exactly!

File: src/lib/fragments.js (hunk 2 above)
Comment (valse) [reply src/lib/fragments.js:13]: Mmm... GraphQl fragments are able to access variables declared in the query... I'll check if they accept arguments too; we could pass the max width, right.

**Pull request merged**